vault-admin-client/src/test/java/com/scality/osis/vaultadmin/impl/BaseTest.java

```diff
@@ -50,11 +62,13 @@ public void init() throws IOException {
 
   protected void initMocks() {
     accountServicesClient = mock(AccountServicesClient.class);
+    stsClient = mock(SecurityTokenServicesClient.class);
 
     initCreateAccountMocks();
     initListAccountsMocks();
+    initAssumeRoleMocks();
 
-    vaultAdminImpl = new VaultAdminImpl(accountServicesClient, adminEndpoint);
+    vaultAdminImpl = new VaultAdminImpl(accountServicesClient, stsClient, adminEndpoint, s3Endpoint);
```

Comment:
Should it be VaultEndpoint?
S3 and vault Endpoint will/can be different(port numbers).
Making it future proof now, will help.

Reply:
Done in 9101950281bb1e469ee5c3cbe333dcaa4c9a3953